src/structures/GuildAuditLogs.js

```diff
@@ -259,7 +259,7 @@ class GuildAuditLogsEntry {
     if (targetType === Targets.UNKNOWN) {
       /**
        * The target of this entry
-       * @type {TargetType}
+       * @type {?Object|Guild|User|Role|Emoji|Invite|Webhook}
```

Comment:
please put this back and change the typedef at the top of the file

Reply:
Shouldn't I instead make a `TargetEntry` or something, since `TargetType` is for, well, the type